src/Query/Builder.php

```diff
@@ -961,8 +962,8 @@ public function where($column, $operator = null, $value = null, $boolean = 'and'
             }
         }
 
-        if (func_num_args() === 1 && is_string($column)) {
-            throw new ArgumentCountError(sprintf('Too few arguments to function %s("%s"), 1 passed and at least 2 expected when the 1st is a string.', __METHOD__, $column));
+        if (func_num_args() === 1 && ! is_array($column) && ! is_callable($column)) {
```

Comment:
Because we want to convert `Model::where('foo', '$gt', 5)` to `Model::where('foo', 'gt', 5)`, but not `Model::where('foo', '$dollar')` which will result in an equality `{ foo: '$dollar' }`

Reply:
Ah, so the signature is really `where($column, $value)` when two arguments are supplied, and `where($column, $operator, $value)` when receiving three arguments. That was lost on me.

I realize this is probably an Eloquent API, but a comment in this method might serve as a helpful reminder.